src/sentry/static/sentry/app/components/globalModal.tsx

```diff
@@ -4,12 +4,28 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Reflux from 'reflux';
 import createReactClass from 'create-react-class';
+import {ModalHeader, ModalBody, ModalFooter} from 'react-bootstrap';
 
 import {closeModal} from 'app/actionCreators/modal';
 import Confirm from 'app/components/confirm';
 import ModalStore from 'app/stores/modalStore';
-
-class GlobalModal extends React.Component {
+import {ModalOptions} from 'app/actions/modalActions';
+
+export type RenderProps = {
```

Comment:
This might be better in `app/actionCreators/modal` since `openModal` is the "public" interface

Reply:
@billyvg updated